precompiles/xvm/src/lib.rs

```diff
@@ -90,9 +92,10 @@
 
         let call_to = input.read::<Bytes>()?.0;
         let call_input = input.read::<Bytes>()?.0;
+        let value = input.read::<Balance>()?;
         let from = R::AddressMapping::into_account_id(handle.context().caller);
 
-        let call_result = XC::call(xvm_context, vm_id, from, call_to, call_input);
+        let call_result = XC::call(xvm_context, vm_id, from, call_to, call_input, value);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new field `value` of type `Balance` is being read from the `input` object. Please ensure that this field is correctly provided in the input and handle any potential errors or validation requirements.
```diff
let value = input.read::<Balance>()?;
let from = R::AddressMapping::into_account_id(handle.context().caller);

let call_result = XC::call(xvm_context, vm_id, from, call_to, call_input, value);
```

Overall, please review the changes carefully to ensure that the modifications to the `xvm_call` function align with the intended behavior and maintain consistency with the rest of the codebase.


<!-- This is an auto-generated comment by OSS CodeRabbit -->